### ISSUE TITLE
Remove unused data from cache

### ISSUE THREAD

**Issue by Nicolabo** [iterative/dvc#820]
I am wondering wether there is a command to delete unused cache directories. For example, I checked 10 combinations, each on separate branch. I chose the last one, and merge with master. But in .dvc/cache I keep data from all branches. Is it possible to delete it somehow?

**Comment by efiop** [comment]
Hi @Nicolabo !

You can use `dvc gc` to remove _all_ cache except the one used in your _current_ branch. Just be sure to account that it will only leave the cache that is used in your current branch, so if you need your cache from other branches too you might want to `dvc checkout mybranch && dvc push` it first, so you have backup for those.

Thanks,
Ruslan

**Comment by efiop** [comment]
Yes, it is an expected behavior since those directories are reused very often. But I agree, it would be cleaner if we removed empty dirs. Created https://github.com/iterative/dvc/issues/821 for that.

Thank you!
